### PR TITLE
added runtime requirements and caveats

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

File: modules/ROOT/pages/cloudhub-setup.adoc [hunk 1]
```diff
@@ -56,12 +56,7 @@ For more information about Anypoint Monitoring, see xref:monitoring::enable-apps
 
 == Verify Supported Mule Versions for Policies
 
-The APIs and policies that Anypoint Visualizer displays in the Policy Visualization come from an update to the agent in Mule runtime engine (Mule). As a result, policies are shown in Anypoint Visualizer only for applications running on these Mule versions:
-
-* Mule 3.9.3+ patch releases available on December 12, 2019 or later.
```
Comment (hannanelson) [inline modules/ROOT/pages/cloudhub-setup.adoc]: I think the "or later" covers the +. Remove the plus?

File: modules/ROOT/pages/cloudhub-setup.adoc [hunk 2]
```diff
@@ -56,12 +56,7 @@ For more information about Anypoint Monitoring, see xref:monitoring::enable-apps
...
-* Support for Mule 4.2.x will be available in the next patch update.
-
-// add! dates! above! dependency
+The APIs and policies that Anypoint Visualizer displays in the Policy visualization come from an update to the agent in Mule runtime engine (Mule). As a result, policies are shown in Anypoint Visualizer only for applications running on any of the patches of Anypoint Visualizer supported Mule versions available on December 12, 2019 or later.
```
Comment (hannanelson) [inline modules/ROOT/pages/cloudhub-setup.adoc]: Should visualization be init cap?
I had to read the 2nd sentence twice. Maybe "on Mule version patches supported by Anypoint Visualizer that were released on xxx and later"?

File: modules/ROOT/pages/policy-visualization.adoc [hunk 3]
```diff
@@ -5,7 +5,7 @@ endif::[]
 
 == About the Policy Visualization
 
-When deploying a Mule app, you must ensure that you are using a version of Mule that supports the Policy visualization. Learn more about which xref:setup.adoc#select-a-deployment-type[runtimes are supported].
+When deploying a Mule app, you must ensure that you are using a version of Mule that supports the Policy visualization. Learn more about which xref:setup.adoc#select-a-deployment-type[runtimes are supported]. The Policy visualization is not supported for customers with API Manager 1.x
```
Comment (hannanelson) [inline modules/ROOT/pages/policy-visualization.adoc]: Do you need "for customers"?

File: modules/ROOT/pages/runtime-fabric-setup.adoc [hunk 4]
```diff
@@ -33,12 +33,7 @@ You must set the property to `false` because the property is disabled by default
 
 == Verify Supported Mule Runtime Engine (Mule) Versions for Policies
 
-The APIs and policies that Anypoint Visualizer displays in the Policy Visualization come from an update to the agent in the Mule runtime. As a result, policies are shown in Anypoint Visualizer only for applications running on these Mule versions:
-
-* Mule 3.9.3+ patch releases available on December 12, 2019 or later.
-* Support for Mule 4.2.x will be available in the next patch update.
-
-// add date! dependency!
+The APIs and policies that Anypoint Visualizer displays in the Policy visualization come from an update to the agent in Mule runtime engine (Mule). As a result, policies are shown in Anypoint Visualizer only for applications running on any of the patches of Anypoint Visualizer supported Mule versions available on December 12, 2019 or later.
```
Comment (hannanelson) [inline modules/ROOT/pages/runtime-fabric-setup.adoc]: See prev

**Review by hannanelson — APPROVED** [review]

File: modules/ROOT/pages/cloudhub-setup.adoc (hunk 2 above)
Comment (hannanelson) [reply modules/ROOT/pages/cloudhub-setup.adoc]: No, that doesn't work either, I see.
I would say add a hyphen after Visualizer (Visualizer-supported), but I think we can't do that.
"Anypoint Visualizer shows policies only for apps running on supported Mule versions released on xxxx and later."

Do customers know how to get the subset of Mule versions that support Visualizer?

File: modules/ROOT/pages/policy-visualization.adoc (hunk 3 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/policy-visualization.adoc]:
```suggestion
When deploying a Mule app, you must ensure that you are using a version of Mule that supports the Policy visualization. Learn more about which xref:setup.adoc#select-a-deployment-type[runtimes are supported]. The Policy visualization is not supported for API Manager 1.x.
```

File: modules/ROOT/pages/cloudhub-setup.adoc (hunk 2 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/cloudhub-setup.adoc]:
```suggestion
The APIs and policies that Anypoint Visualizer displays in the Policy visualization come from an update to the agent in Mule runtime engine (Mule). As a result, Anypoint Visualizer shows policies only for apps running on supported Mule versions released on December 12, 2019 or later.
```

File: modules/ROOT/pages/runtime-fabric-setup.adoc (hunk 4 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/runtime-fabric-setup.adoc]:
```suggestion
The APIs and policies that Anypoint Visualizer displays in the Policy visualization come from an update to the agent in Mule runtime engine (Mule). As a result, Anypoint Visualizer shows policies only for apps running on supported Mule versions released on December 12, 2019 or later.
```

**Pull request merged**